transport/filter.go

```diff
@@ -76,6 +76,10 @@ func (fo filteredOutbound) Stop() error {
 	return fo.o.Stop()
 }
 
+func (fo filteredOutbound) Options() Options {
+	return fo.o.Options()
```

Comment:
ty sir - this prevents breaking the Law of Demeter by disallowing deep access to `fo.o.Options()` from the outside.